src/control/lib/control/network.go

```diff
@@ -216,14 +217,15 @@ type (
 	ClientNetworkHint struct {
 		// These CaRT settings are shared with the
 		// libdaos client to aid in CaRT initialization.
-		Provider    string   `json:"provider"`
-		Interface   string   `json:"interface"`
-		Domain      string   `json:"domain"`
-		CrtTimeout  uint32   `json:"crt_timeout"`
-		NetDevClass uint32   `json:"net_dev_class"`
-		SrvSrxSet   int32    `json:"srv_srx_set"`
-		EnvVars     []string `json:"env_vars"`
-		ProviderIdx uint32   `json:"provider_idx"`
+		Provider           string   `json:"provider"`
+		Interface          string   `json:"interface"`
+		Domain             string   `json:"domain"`
+		CrtTimeout         uint32   `json:"crt_timeout"`
+		NetDevClass        uint32   `json:"net_dev_class"`
+		SrvSrxSet          int32    `json:"srv_srx_set"`
+		EnvVars            []string `json:"env_vars"`
+		ProviderIdx        uint32   `json:"provider_idx"`
+		ClientFirewallMode bool     `json:"client_firewall_mode"`
```

Comment:
I wouldn't block on it but I think it's easier for people to follow if the field name matches the flag used in the command. This should generally be followed unless there is a specific reason not to IMO.